src/test/java/org/takes/rq/multipart/RqMtFakeTest.java

```diff
@@ -43,40 +47,30 @@
  * Test case for {@link RqMtFake}.
  * @since 0.33
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
- * @checkstyle MultipleStringLiteralsCheck (500 lines)
  */
+@SuppressWarnings({"PMD.AvoidDuplicateLiterals", "PMD.TooManyMethods"})
 public final class RqMtFakeTest {
-    /**
-     * Form data.
-     */
-    private static final String FORM_DATA =
```

Comment:
@Serranya Why remove the fields `FORM_DATA` and `DISPOSITION`? I think it's better to leave them and get rid of `PMD.AvoidDuplicateLiterals` supression above, WDYT?

Reply:
I just didn't like the shared constants between the tests. I will add them again.

Reply:
@paulodamaso I agree with @Serranya. I prefer don't share anything between tests, acoording with [it](https://www.yegor256.com/2016/05/03/test-methods-must-share-nothing.html).

Reply:
@fabriciofx @Serranya I've read the article, and I totally agree with it, but I think that we are talking about different things here. The article talks about not sharing constants between tests, but the constants showed in the article are domain values that are being tested. In our case we have some fixed value that is a standard set by the HTTP spec, so it'll always be the same in all the test cases no matter what happens. Could you see the difference between the two situations?

Reply:
@paulodamaso I see. So we can keep a constant for `Content-Disposition` and `Content-Length`, but not for the actual content of the header field (The previous `FORM_DATA`).

Reply:
@Serranya Yes, I think that would be enough. Actually we should have these constants defined somewhere else, like we have for `HTTP_STATUS`, but I don't know yet where put them.

Reply:
@paulodamaso @Serranya I agree in parts. IMHO, we must not share objects between tests, but I agree there are some repetitions that can be avoided. So, let's do as Yegor's post show us: put the repetition string in each test's local variable method and share this variable at own test. I think it's a happy medium.

Reply:
@fabriciofx I did what you suggested locally. I think the tests are now way less readable then with the repeated strings.

If you still want to go ahead with that approach I will push my changes. But I wanted to ask you to reconsider.

I think the tests with the repeated Strings are fine now. I can add a puzzle that asks for a central definition of the Header Fields.

Reply:
@Serranya I think that we are wasting too much time here, so please keep `Content-Disposition` and `Content-Length` as constants in this class and add a puzzle so we can put these as domain-related constants somewhere else later, just like `HttpURLConnection.HTTP_OK` and `RqMethod.POST`

Reply:
@paulodamaso Ok. Will do

Reply:
@Serranya I agree with @paulodamaso. You can go ahead with the changes. Sorry for this rework!